src/main/thrift/octopus.thrift

```diff
@@ -0,0 +1,88 @@
+namespace scala com.gu.flexibleoctopus.model.thrift
+
+/** date times are reprsented as i64 - epoch millis */
+typedef i64 DateTime
+
+/** 
+* A single story bundle that maps directly to a piece of content in Composer 
+*/
+struct StoryBundle {
+
+    1: required i64 id;
+
+    2: required string composerId;
+
+    3: required string publicationCode;
+
+    4: required string sectionCode;
+
+    5: required Article bodyText;
+
+    6: optional Article headline;
+
+    // the primary page that the article is on 
+    7: optional i64 pageNumber;
+
+    // the issue date, this is midnight UTC of the date of publication
+    8: optional DateTime printPublicationDate;
```

Comment:
I have a better plan!!! TIL that as well as epoch millis and epoch seconds there are also epoch days. We should use epoch days to store this. Java's [LocalDate](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html) has `toEpochDay` and `ofEpochDay` so this is properly supported and _not a hack_™️.

We can have a new `typedef i64 Date` to represent this.